### DISCUSSION TITLE
SOLVED: Running this on a local server with SteamCMD?

### DISCUSSION THREAD

**Discussion by Sketaful** [BepInEx/BepInEx#528] [Help and Q&A]
I don't know what I need to do to make this work.
I got it running on on my client without any problem but it refuses to even get a reaction from the server.

I've tried google to find where to put the files to make it work on a server but the really few guides I found that wasn't about *inserthostedsitehere* was about just running a dedicated server on your gaming machine by installing the dedicated server in steam.

How do I do when I have a separate locale machine running the server with SteamCMD?

The script for starting the server atm is a .bat-file containing:
c:\server\valheim\valheim_server -nographics -batchmode -name "SERVERNAME" -port 2500 -password "PASSWORD" -world "WORLDNAME" -savedir "C:\Server\Valheim-Serverworld" -crossplay

(Servername, password and worldname are of course something else. :)

EDIT: Running a local WINDOWS machine I mean. :)

EDIT: SOLVED! 
Finally, after ages of searching I find it within 5 min from posting the question here. Oh well. Someone just remove/close this. :)

**Comment by beccsjb** [comment]
Would you mind referencing the found solution please?

**Reply by Sketaful** [reply]
6 month later and at work so need to check when I get home. But I "think" the main problem was me not using the bepinex made for valheim and instead downloaded it fresh from here.

I 'think' I just used the one here: https://valheim.thunderstore.io/package/denikson/BepInExPack_Valheim/
